src/Microplate/Scalars/Row16MM.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\Microplate\Scalars;
+
+use MLL\GraphQLScalars\Regex;
+
+class Row16MM extends Regex
+{
+    public ?string $description = 'Represents a row in a coordinate system with 16 rows without the letter J (see AE-347)';
```

Comment:
```suggestion
    public ?string $description = 'Represents a row in a coordinate system with 16 rows without the letter J';
```
Keine Referenz zu internen Tickets